Libplanet/Net/Swarm.cs

```diff
@@ -450,27 +455,22 @@ IEnumerator IEnumerable.GetEnumerator()
             return GetEnumerator();
         }
 
-        public async Task StartAsync<T>(
-            BlockChain<T> blockChain,
```

Comment:
This and other methods that had taken a `blockChain` argument should be addressed in the changelog.  Not only that it no more takes the argument, but also that it became no more a generic method.

Reply:
Added a description of the methods.